IanGRap/Squish
# Audio manager updates

Contributors: Alfredo

Goal: Add a new function that will choose from an array of Block Break Sounds (provided by Fernando) and pitch shift them slightly every time they are called. 

Changes: 
* AudioManager.cs: Updated the class to use a Dictionary when searching for songs. Added a new function that calls VariableSoundPitchShifter.cs
* VariableSoundPitchShifter.cs: This takes a list of block break sounds. It has a public function that will PlayOneShot() a random block break sound and pitch shift it slightly. 
* CameraManager.cs, PlayerSelectBlockSpawner.cs: these have been changed to call the new PlayOneShotBlockBreak()
* BlockDamager: This used to call Play("") but it has been changed to PlayOneShotBlockBreak()

How to test: 
* Play through all parts of the game. Make sure that the block breaking sounds are ok. Try to listen for the slight changes in the block breaking sounds. 
* After you have done this Play the game but this time go into the AudioManager and in VariableSoundPitchShifter change the range of Min and Max. You should notice a more drastic pitch shift. 1== normal pitch.
----
Implemented a suggested change to make a class to hold the info needed for any new sound groups
